### PR TITLE
COM-20065: Implement card view for bundle list

### PR DESCRIPTION
**Pull request by ciastektk**
https://jira.ez.no/browse/COM-20065

bundles ez platform 4

### CODE REVIEW

File: app/Resources/views/themes/ezplatform/line/bundle_card.html.twig [hunk 1]
```diff
@@ -1,54 +1,29 @@
...
-            <div class="bundle-card-line-authors">
-                <div class="bundle-card-line-owner">
-                    {% if packageDetails.authorAvatarUrl is defined and packageDetails.author[0] is defined and packageDetails.author[0].name is defined %}
-                        <span>Owner:</span>
+    <a href="{{ ez_field_value(content, 'packagist_url') }}" title="See on Packagist.org">
+        <div class="bundle-card-line">
+            <div class="bundle-card-line-data">
+                <div class="bundle-card-line-authors">
+                    <div class="bundle-card-line-owner">
+                        {% if packageDetails.authorAvatarUrl is defined and packageDetails.author is defined and packageDetails.author.name is defined %}
```
Comment (SylvainGuittard) [inline app/Resources/views/themes/ezplatform/line/bundle_card.html.twig]: Why not?
```suggestion
                        {% if packageDetails.authorAvatarUrl is defined and packageDetails.author.name is defined %}
```

File: app/Resources/views/themes/ezplatform/line/bundle_card_old.html.twig [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<div class="bundle-card-line-container{% if extra.page is defined %} page-{{ extra.page }}{% endif %}">
```
Comment (SylvainGuittard) [inline app/Resources/views/themes/ezplatform/line/bundle_card_old.html.twig]: I don't think this file should be in this PR ;)

Comment (SylvainGuittard) [reply app/Resources/views/themes/ezplatform/line/bundle_card_old.html.twig]: This is why we have git. So we can restore the previous version of the file if we need to.
There is no need to keep an unused file in the repo.

Comment (ciastektk) [reply app/Resources/views/themes/ezplatform/line/bundle_card_old.html.twig]: Ok, I removed this template

File: web/assets/scss/bundle.scss [hunk 3]
```diff
@@ -1,41 +1,37 @@
...
-    padding: 0 45px 0 20px;
 }
 
 .bundle-card-line-information h2 {
-    font-size: 26px;
-    color: #63605d;
-    font-weight: 400;
+    font-size: 19px;
+    color: #333;
+    font-weight: bold;
```
Comment (SylvainGuittard) [inline web/assets/scss/bundle.scss]: I think it's better to be consistent with the rest of the css and not use `bold`.
```suggestion
    font-weight: 600;
```

**Pull request merged**